src/libsync/utility.cpp

```diff
@@ -275,7 +275,7 @@ void Utility::usleep(int usec)
 
 bool Utility::fsCasePreserving()
 {
-#ifndef WITH_TESTING
+#ifdef WITH_TESTING
```

Comment:
heh

Reply:
I did it wrong initially and it worked since the right config.h wasn't included here.